ledger/src/blockstore_processor.rs

```diff
@@ -1146,25 +1155,32 @@ impl BatchExecutionTiming {
 
         saturating_add_assign!(*wall_clock_us, new_batch.execute_batches_us);
 
-        use ExecuteTimingType::{NumExecuteBatches, TotalBatchesLen};
-        totals.saturating_add_in_place(TotalBatchesLen, new_batch.total_batches_len);
-        totals.saturating_add_in_place(NumExecuteBatches, 1);
+        use ExecuteTimingType::NumExecuteBatches;
+        // These metrics aren't applicable for the unified scheduler
+        if !is_unified_scheduler_enabled {
+            use ExecuteTimingType::TotalBatchesLen;
```

Comment:
what's up with these `use`? why not put them in the top-of-file list, or at least in the same block here.

Reply:
done: d41276a2c64428a8b4e326a8816e0afbc67d26b0